mmv1/third_party/terraform/services/compute/resource_compute_backend_service_test.go.tmpl

```diff
@@ -146,6 +146,40 @@ func TestAccComputeBackendService_withBackendAndIAP(t *testing.T) {
 	})
 }
 
+func TestAccComputeBackendService_withBackendAndPreference(t *testing.T) {
+	t.Parallel()
+
+	serviceName := fmt.Sprintf("tf-test-%s", acctest.RandString(t, 10))
+	igName := fmt.Sprintf("tf-test-%s", acctest.RandString(t, 10))
+	itName := fmt.Sprintf("tf-test-%s", acctest.RandString(t, 10))
+	checkName := fmt.Sprintf("tf-test-%s", acctest.RandString(t, 10))
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeBackendServiceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeBackendService_withBackendAndPreference(
+					serviceName, igName, itName, checkName, "DEFAULT", 10),
+			},
+			{
+				ResourceName:      "google_compute_backend_service.lipsum",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeBackendService_withBackendAndPreference(
+					serviceName, igName, itName, checkName, "PREFERRED", 20),
```

Comment:
It looks like serviceName, igName, itName, checkName are not updated in this step and are the same with the previous step. Those names don't need to be passed as parameters.

Reply:
Done